packages/select/test/queryListTests.tsx

```diff
@@ -119,6 +129,59 @@ describe("<QueryList>", () => {
                 query: "123",
             });
             assert.deepEqual(filmQueryList.state().activeItem, TOP_100_FILMS[1]);
+
+            filmQueryList.unmount();
+        });
+    });
+
+    describe("activeItem state initialization", () => {
+        it("initializes to first filtered item when uncontrolled", () => {
+            const props: IQueryListProps<IFilm> = {
+                ...testProps,
+                // Filter down to only item at index 11, so item at index 11 should be
+                // chosen as default activeItem
+                itemPredicate: (_query, item) => item === TOP_100_FILMS[11],
+                query: "123",
+            };
+
+            const filmQueryList: ReactWrapper<IQueryListProps<IFilm>, IQueryListState<IFilm>> = mount(
```

Comment:
DRY! define a type alias at the top of this file: `type QueryListWrapper = ReactWrapper<IQueryListProps<IFilm>, IQueryListState<IFilm>>`

Reply:
I've been in the habit of always unmounting. I seem to remember enzyme documentation recommending always unmounting, but current documentation says: "Keep that in mind while writing your tests and, if necessary, use .unmount() or something similar as cleanup."

I can remove the unmount calls here since they technically aren't necessary, but I'll continue to always unmount as a rule in my projects so that I don't get obscure problems in the situations where unmounting actually is important.

Reply:
done